api/user_test.go

```diff
@@ -199,14 +201,48 @@ func TestLogin(t *testing.T) {
 		Password:    "passwd1",
 		AuthService: model.USER_AUTH_SERVICE_LDAP,
 	}
-	user3 = Client.Must(Client.CreateUser(user3, "")).Data.(*model.User)
-	store.Must(th.App.Srv.Store.User().VerifyEmail(user3.Id))
+	ruser3 := Client.Must(Client.CreateUser(user3, "")).Data.(*model.User)
+	store.Must(th.App.Srv.Store.User().VerifyEmail(ruser3.Id))
 
-	if _, err := Client.Login(user3.Id, user3.Password); err == nil {
+	if _, err := Client.Login(ruser3.Id, user3.Password); err == nil {
```

Comment:
Prior to this change, the `err` returned was `Password field must not be blank`, which obviously doesn't test the assertion in question.

Reply:
I think either way is fine. Consistency isn't as important here since these tests are for the old API which has been deprecated and will be removed come 5.0